src/components/form/advanced-form-inputs/radio.jsx

```diff
@@ -0,0 +1,29 @@
+import { forwardRef } from "react";
+
+/**
+ * The following is a Radio component that is compatible with react-hook-form
+ * @param {string|undefined} label - Desired label for form input
+ * @param {string} name - Should be set by the register function of react-hook-form.
+ * @param {string} value - value for radio button
+ * @param {*} rest - Register props from react-hook-form
+ * @param {*} ref - the ref passed by react-hook-form register function
+ */
+const Radio = forwardRef(({ label, value, name, ...rest }, ref) => {
+  return (
+    <label
```

Comment:
Right now, clicking on the label associated with a radio button won't select the radio input itself. While maybe not by definition wrong, it is a pet peeve of mine :) one way to fix this would be to line up the `label`'s `for` attribute with the `input`'s `id` attribute, [see here](https://stackoverflow.com/a/6293626/11646872) for more information.